pkg/rules/nacos/service_holder/nacos_go_client_service_holder_setup.go

```diff
@@ -57,11 +57,7 @@ func afterNewServiceInfoHolder(call api.CallContext, holder *naming_cache.Servic
 			Key:   "not.load.cache.at.start",
 			Value: attribute.StringValue(call.GetKeyData("notLoadCacheAtStart").(string)),
 		})
-		count := 0
-		holder.ServiceInfoMap.Range(func(k, v interface{}) bool {
-			count++
-			return true
-		})
+		count := holder.ServiceInfoMap.Count()
```

Comment:
@123liuziming Please help review nacos-rule changes, thanks.

ServiceInfoHolder.ServiceInfoMap is type of [`cache.ConcurrentMap`](https://github.com/nacos-group/nacos-sdk-go/blob/v2.0.0/clients/cache/concurrent_map.go), it doest implement Range method.